File: CCC 2011/WhichAlien.java
import java.util.*;

public class WhichAlien {
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        System.out.println("How many antennas?");
        int antennas = Integer.parseInt(in.nextLine());
        System.out.println("How many eyes?");
        int eyes = Integer.parseInt(in.nextLine());

        if (antennas >= 3 && eyes <= 4){
            System.out.println("TroyMartian");
        }
        if (antennas <= 6 && eyes >= 2){
            System.out.println("VladSaturnian");
        }
        if (antennas <= 2 && eyes <= 3){
            System.out.println("GraemeMercurian");
        }

    }
}

File: CCC 2011/WhichAlien.cpp
#include <iostream>

using namespace std;

void isToryMartian(int A, int E){
    if (A >= 3 && E <= 4){
        cout << "TroyMartian\n" << endl;
    }
}
void isVladSaturnian(int A, int E){
    if (A <= 6 && E >= 2){
        cout << "VladSaturnian\n" << endl;
    }
}
void isGraemeMercurian(int A, int E){
    if (A <= 2 && E <= 3){
        cout << "GraemeMercurian\n" << endl;
    }
}
int main()
{
    int A, E;
    cout << "How many antennas?" << endl;
    cin >> A;
    cout << "How many eyes?" << endl;
    cin >> E;
    isToryMartian(A, E);
    isVladSaturnian(A, E);
    isGraemeMercurian(A, E);
    return 0;
}